### PR TITLE
Change the jwt secret property to the base64 secret where it was missing

### PR DESCRIPTION
**Pull request by Akuka**
-   Please make sure the below checklist is followed for Pull Requests.

-   [ ] [Travis tests](https://travis-ci.org/jhipster/generator-jhipster/pull_requests) are green
-   [ ] Tests are added where necessary
-   [ ] Documentation is added/updated where necessary
-   [ ] Coding Rules & Commit Guidelines as per our [CONTRIBUTING.md document](https://github.com/jhipster/generator-jhipster/blob/master/CONTRIBUTING.md) are followed

### CODE REVIEW

**Comment by jdubois** [comment]
Is there a specific reason to change this? At the moment we stick to the less secure non-base64 version for people doing microservices with a mix of older and newer JHipster applications (as the older ones don't use this property).
We will migrate to the base64 version in a few months.

**Comment by jdubois** [comment]
Oops sorry @Akuka I'm currently doing a microservice and we are indeed with the base64 property... So I don't know the state of this!!! So please ignore my comment, but can you explain why you want to modify this, as if I'm lost I guess I'm not the only one (hopefully...)

**Comment by Akuka** [comment]
Not a specific issue, just to be aligned with the rest of the generated code/configuration 
related changes: https://github.com/jhipster/generator-jhipster/issues/8165 and https://github.com/jhipster/generator-jhipster/commit/416a62449c77d9c9b6e9da1f81fc6a1315b721e5

File: generators/entity-server/templates/src/main/java/package/common/inject_template.ejs [hunk 1]
```diff
@@ -39,7 +39,7 @@
     }
 _%>
 
-    <%= beans.map(bean => `private final ${bean.class} ${bean.instance};`).join('\n\n    ') %>
+    <%= beans.map(bean => `protected final ${bean.class} ${bean.instance};`).join('\n\n    ') %>
```
Comment (jdubois) [inline generators/entity-server/templates/src/main/java/package/common/inject_template.ejs]: This is from another commit, and I know we must discuss this, but we can't merge this with this PR anyway

Comment (jdubois) [reply generators/entity-server/templates/src/main/java/package/common/inject_template.ejs]: I'm correcting this and trying to merge....

**Comment by jdubois** [comment]
Merged, thanks @Akuka !

**Pull request merged**